core/src/main/scala/kafka/api/ApiVersion.scala

```diff
@@ -84,6 +84,8 @@ object ApiVersion {
   private val versionMap: Map[String, ApiVersion] =
     allVersions.map(v => v.version -> v).toMap ++ allVersions.groupBy(_.shortVersion).map { case (k, v) => k -> v.last }
 
+  val allVersionsArray = versionMap.keys.seq.toArray
```

Comment:
nit: this might be more generally useful if exposed as a Set. We can convert it to a seq or an array when we pass it to `in` in the config definition.